### PR TITLE
fix #3 🍱 add project assets

### LINKED ISSUE #3: Set up Project
acceptance criteria 

- [ ] configurate global css
- [ ] add all assets to public folder

### PR DESCRIPTION
**Pull request by RonjaMeinecke**
• add all assets

### CODE REVIEW

File: public/assets/BgBlobYellow.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg id="BgBlobYellow" xmlns="http://www.w3.org/2000/svg" width="483.277" height="327.657" viewBox="0 0 483.277 327.657">
+  <path id="Pfad_164" data-name="Pfad 164" d="M34.438,34.434C-51.328,62.6,50.2,121.652,50.227,219.152s56.119,73.125,270.111,47.893C476.98,248.576,486.24,38.934,382.055,3.3,343.971-9.725,80.5,20.178,34.438,34.434Z" transform="translate(458.059 327.75) rotate(-175)" fill="#ffffd6"/>
+  <g id="Pfad_165" data-name="Pfad 165" transform="translate(450.747 317.472) rotate(-177)" fill="none">
```
Comment (mewiemers) [inline public/assets/BgBlobYellow.svg:3]:
```suggestion
  <g id="Pfad_165" transform="translate(450.747 317.472) rotate(-177)" fill="none">
```

**Review by mewiemers — COMMENTED** [review]
;-)

**Review by cboenke — APPROVED** [review]
LGTM 🚀

**Review by mewiemers — APPROVED** [review]
looks like a beautiful tomatoe 🍅

**Pull request merged**